classes/OccurrenceManager.php

```diff
@@ -637,10 +637,8 @@ public function getQueryTermStr(){
 			if($v) $retStr .= '&'. $this->cleanOutStr($k) . '=' . $this->cleanOutStr($v);
 		}
 		if(isset($this->taxaArr['search'])){
-			$patternOfOnlyLettersDigitsAndSpaces = '/^[a-zA-Z0-9\s\-\,]*$/'; // TOOD accommodate symbols associated with extinct taxa, hybrid crosses, and abbreviations with periods, e.g. "var."?
-
-
-			if (preg_match($patternOfOnlyLettersDigitsAndSpaces, $this->getTaxaSearchTerm())==1) {
+			$patternTaxonChars = '/^[a-zA-Z0-9\s\-\,\.×†]*$/';
+			if (preg_match($patternTaxonChars, $this->getTaxaSearchTerm())) {
```

Comment:
I think we needed the ==1 explicitly here because the way PHP handles falsy is different from the way javaScript handles it. Right, @GregPost-ASU ? Didn't we play around with this a bit and conclude that this was the safest way to do it?

Reply:
@Atticus29  - Yes, I recall us discussing this.  After reviewing the documentation on preg_match (https://www.php.net/manual/en/function.preg-match.php) I believe Ed's code will have the same result.  I also tested this code locally and it does resolve the download issue.